.github/workflows/lint_pr_format.yml

```diff
@@ -24,6 +36,6 @@ jobs:
       - uses: jesusvasquez333/verify-pr-label-action@v1.4.0
```

Comment:
maybe is worth extracting this in its own job ? 
```
jobs: 
  check_title: ....
  check_labels: .... 
  check_project: ... # ofc to be added at a later point
```

Reply:
Yes, of course. I didn't even noticed that this was in "check_title" until you've pointed it out 😄

Done in 3b3ade64c0